Doc/library/socket.rst

```diff
@@ -182,8 +182,10 @@ created.  Socket addresses are represented as follows:
   ``(ifname, proto[, pkttype[, hatype[, addr]]])`` where:
 
   - *ifname* - String specifying the device name.
-  - *proto* - An in network-byte-order integer specifying the Ethernet
-    protocol number.
+  - *proto* - The Ethernet protocol number. May be one of
+    :ref:`ETHERTYPE_* constants <socket-ethernet-types>`
+    or any other Ethernet protocol number.
+    In both cases, values must be in network-byte-order.
```

Comment:
```suggestion
    Value must be in network-byte-order.
```

* Shorter and clearer
* More than two cases now
* At least per the description, this only has a single value, not multiple